prow/plugins/trigger/pr.go

```diff
@@ -49,7 +49,27 @@ func handlePR(c client, trustedOrg string, pr github.PullRequestEvent) error {
 				return fmt.Errorf("could not welcome non-org member %q: %v", author, err)
 			}
 		}
-	case github.PullRequestActionReopened, github.PullRequestActionSynchronize:
+	case github.PullRequestActionReopened:
+		// When a PR is reopened, check that the user is in the org or that an org
+		// member had said "/ok-to-test" before building.
+		comments, err := c.GitHubClient.ListIssueComments(pr.PullRequest.Base.Repo.Owner.Login, pr.PullRequest.Base.Repo.Name, pr.PullRequest.Number)
```

Comment:
Why don't we do the self-membership check first and avoid this list plus any commentors auth checks?